tools/set_user_role.py

```diff
@@ -0,0 +1,123 @@
+import argparse
+import sys
+
+parser = argparse.ArgumentParser(
+    description='Elevate user to super admin', add_help=True
+)
+parser.add_argument('--username', help='User to set role for')
+parser.add_argument('--list', action='store_true', help='List all users')
+parser.add_argument('--role', help='Role to elevate user to')
```

Comment:
```suggestion
parser.add_argument('--role', help='Role to elevate user to')
parser.add_argument('--list', action='store_true', help='List all users')
```

Just swap these for consistency with typical usage.